packages/docs/modules/page-config/blocks/file/transform.ts

```diff
@@ -1,13 +1,15 @@
 import { defineBlockTransform } from "../../compiler/define-block-transform";
 import { extname } from 'path'
+import { readFile } from 'fs/promises'
 
 export default defineBlockTransform(async function (block) {
   if (block.type !== 'file') { return }
 
   const importName = block.args[0].slice(1, -1)
   const importPath = (await this.importer.resolveAbsolutePath(`${importName}`))!
-  const importComponent = this.importer.importDefault('file', `${importPath}?raw`)
+  const content = (await readFile(importPath, 'utf-8')).toString()
```

Comment:
Add at least `?` before `toString` or any typo will lead to page down.

Reply:
I'll throw human-readable error. It's okay to have Error in transform functions, so vite can handle files correctly.